spring-boot/pom.xml

```
@@ -37,4 +60,4 @@
 			</plugin>
 		</plugins>
 	</build>
-</project>
+</project>
```

* https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b
* https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log
```suggestion
</project>

```